benchmarks/inclusion/doc/inclusion.md

```diff
@@ -8,57 +8,25 @@ aligns to cell boundaries, leading to much larger difficulties in obtaining an
 accurate representation of the pressure. Specifically, the almost
 discontinuous pressure along this interface leads to oscillations in the
 numerical solution. This can be seen in the visualizations shown in
```

Comment:
Could you link to the SolCx benchmark in line 6? I know that wasn't done previously, but the statement "discussed above" doesn't really make much sense anymore in an html document.